trio/socket.py

```diff
@@ -44,24 +100,17 @@
 except ImportError:
     pass
 
-# expose all uppercase names from standardlib socket to trio.socket
-import socket as _stdlib_socket
-
-globals().update(
-    {
-        _name: getattr(_stdlib_socket, _name)
-        for _name in _stdlib_socket.__dict__ if _name.isupper()
-    }
-)
-
 if _sys.platform == 'win32':
     # See https://github.com/python-trio/trio/issues/39
     # Do not import for windows platform
     # (you can still get it from stdlib socket, of course, if you want it)
     del SO_REUSEADDR
 
 # get names used by trio that we define on our own
-from ._socket import IPPROTO_IPV6
+try:
+    IPPROTO_IPV6
+except NameError:
+    from ._socket import IPPROTO_IPV6
```

Comment:
Again... this code should work fine, but the old code also worked fine, and was 4x shorter and had fewer branches to test. Why change it?

Reply:
I am not sure I understood. I wrapped it because the CI pipeline failed on the double declaration. 
What should I do instead?